src/views.rs

```diff
@@ -111,6 +111,21 @@ where
         };
 
         let draw_day = |printer: &Printer| {
+            let offset = match NaiveDate::from_ymd_opt(year, month, 1) {
```

Comment:
This should probably take into account that in some countries Sunday is the first day of the week, e.g. USA, Canada, Japan. Another thing could be making this offset absolutely optional.

Also, maybe calculating day offset deserves to be in a separate function?

Reply:
> This should probably take into account that in some countries Sunday is the first day of the week, e.g. USA, Canada, Japan. Another thing could be making this offset optional.

Yeah … pretty much what I wrote in the PR description.